README.md

```diff
@@ -40,3 +42,31 @@ isochrone(location, time, resolution, network, function(err, drivetime) {
 ```
 
 Alternativaly the `network` parameter can be an [OSRM](https://github.com/Project-OSRM/node-osrm) module instance. Allowing setup an OSRM with custom paramters, e.g. usage of shared-memory.
+
+You can too define your own function to draw line/polygon instead of default:
+
+```js
+var concave = require('turf-concave');
+var Isochrone = require('osrm-isochrone');
+
+var resolution = 25; // sample resolution
+var time = 300; // 300 second drivetime (5 minutes)
+var maxspeed = '70'; // in 'unit'/hour
+var unit = 'miles'; // 'miles' or 'kilometers'
+var network = './dc.osrm' // prebuild dc osrm network file
+var location = [-77.02926635742188,38.90011780426885]; // center point
+
+var isochrone = new Isochrone(location, time, resolution, maxspeed, unit, network, function(err, drivetime) {
+  if(err) throw err;
+  // your geojson from draw overload
+  console.log(JSON.stringify(drivetime))
+});
+isochrone.draw = function(destinations) {
```

Comment:
The canonical way to handle this case would be an `options` parameter. In this case I think a breaking API change would be fine, because it simplifies the interface radically.